session/io/io.go

```diff
@@ -0,0 +1,594 @@
+package io
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
```

Comment:
`buildkit/session/io` should be renamed to something like `buildkit/session/sessionio`

Reply:
Fixed.